### DISCUSSION TITLE
SpatiaLite Support

### DISCUSSION THREAD

**Discussion by fpichardom** [sqlitecloud/discussions#11] [Feature request] [answered]
Will the service support SQLite spatial extension SpatiaLite? It is a PostGIS alternative for SQLite and essential for geospatial projects

**Comment by marcobambini** [comment, marked as answer]
Hello, @fpichardom, we'll add support for extensions in about 4 weeks and certainly offer SpatiaLite, among many others.
Thanks a lot for your feedback!
